Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:19.10
+FROM ubuntu:latest
```

Comment:
`19.10` and `20.10` fail due to missing repositorys

Reply:
Given that `latest` = LTS, I agree it's a good change.